Answers.md

```diff
@@ -1,5 +1,38 @@
 1.  Explain the differences between `client-side routing` and `server-side routing`.
+
+Server-Side Routing (Standard)
+
+Needs to refresh with every new request. The information requested is the will be the only information given.  This can make things slow if a lot of information Is being requested. 
+
+Client-Side Routing 
+
+Routing handled internally by the JS that is already on the page.  This is the modern way of routing. No refreshing at all and will display data on the drop of a dime because it is already stored. 
+
+Main difference is that server-side requires refresh/reload and Client side doesn't. 
+
+
 1.  What does HTTP stand for?
+ HyperText Transfer Protocol.
+HTTP is the network protocol (set of rules) used by web applications.
+
 1.  What does CRUD stand for?
+Create, Read, Update, Delete. 
+
+
+
 1.  Which HTTP methods can be mapped to the CRUD acronym that we use when interfacing with APIs/Servers.
-1.  Mention three tools we can use to make AJAX requests
\ No newline at end of file
+GET
+The GET method requests a representation of the specified resource. Requests using GET should only retrieve data.
+POST
+The POST method is used to submit an entity to the specified resource, often causing a change in state or side effects on the server.
+DELETE
+The DELETE method deletes the specified resource.
+PUT
+The PUT method replaces all current representations of the target resource with the request payload.
+
+
+
+
+1.  Mention three tools we can use to make AJAX requests
+.then, .catch, .get
```

Comment:
We were looking for tools like Axios, or jquery's Ajax library, or the fetch API.